python_scripts/version.py

```diff
@@ -11,6 +11,9 @@ def analysis_version_string():
     # assuming it is in the git repository
     old_dir = os.getcwd()
     os.chdir(os.path.dirname(os.path.abspath(__file__)))
-    version_string = subprocess.check_output(["git", "describe"]).rstrip('\n')
+    try:
+        version_string = subprocess.check_output(["git", "describe"]).rstrip('\n')
+    except:
```

Comment:
This would avoid catching unrelated exceptions:
```suggestion
    except subprocess.CalledProcessError:
```

Reply:
Thanks for the hint. Will add it accordingly.